packages/chain/contracts/Eth_broker.sol

```diff
@@ -361,7 +361,7 @@ contract Eth_broker {
             msg.sender, 
             _tokenAmount, 
             daiReward, 
-            ethMin, 
+            swapOutputs[1], 
```

Comment:
I find this a very strange change and don't understand why you pass `0` here (the array is initialized but never filled)

Reply:
The outputs from the uniswap broker return the number required here in `swapOutputs[1]`